pkg/plugins/runtime/k8s/metadata/annotations.go

```diff
@@ -11,6 +11,8 @@ import (
 
 // Annotations that can be used by the end users.
 const (
+	// Deprecated: use KumaMeshLabel from pkg/plugins/runtime/k8s/metadata/labels.go instead.
```

Comment:
nit pick: 

```suggestion
	// Deprecated: use KumaMeshLabel as a label from pkg/plugins/runtime/k8s/metadata/labels.go instead of this annotation.
```